internal/services/disks/disk_pool_iscsi_target_lun_resource_test.go

```diff
@@ -303,10 +309,6 @@ resource "azurerm_disk_pool" "test" {
   tags = {
     "env" = "qa"
   }
-  timeouts {
```

Comment:
default timeout for `azurerm_disk_pool` 30m is enough for it. Only other resources require more create/delete time